components/script/dom/webidls/WebGPU.webidl

```diff
@@ -66,11 +68,15 @@ interface GPUAdapterInfo {
 interface mixin NavigatorGPU {
     [SameObject, Pref="dom.webgpu.enabled", Exposed=(Window /* ,DedicatedWorker */)] readonly attribute GPU gpu;
 };
-// NOTE: see `Navigator.webidl`
-// Navigator includes NavigatorGPU;
+Navigator includes NavigatorGPU;
 // NOTE: see `WorkerNavigator.webidl`
 // WorkerNavigator includes NavigatorGPU;
 
+[Exposed=DedicatedWorker]
+partial interface WorkerNavigator {
+    [SameObject, Pref="dom.webgpu.enabled"] readonly attribute GPU gpu;
+};
```

Comment:
[per spec](https://github.com/w3c/webref/blob/9085aef35ca56e26f96d45a5cbd92ada2718c311/ed/idl/webgpu.idl#L67C1-L71C39) this should be:
```idl
interface mixin NavigatorGPU {
    [SameObject, Pref="dom.webgpu.enabled"] readonly attribute GPU gpu;
};
Navigator includes NavigatorGPU;
WorkerNavigator includes NavigatorGPU;
```